### PR TITLE
Updates from v1.9

### PR DESCRIPTION
**Pull request by mikenairn**
## Additional Information
Cherry picking master relevant changes from v1.9 https://github.com/integr8ly/installation/pull/1420

## Verification Steps
N/A

## Checklist

- [ ] Updated [Changelog](https://github.com/integr8ly/installation/blob/master/CHANGELOG.md)
- [ ] Tested Installation
- [ ] Tested Uninstallation
- [ ] Tested or Created follow on for Upgrade

Is an upgrade task required and are there additional steps needed to test this?
- [ ] Yes
- [ ] No

### CODE REVIEW

File: scripts/upgrade.template.yml [hunk 1]
```diff
@@ -63,5 +63,10 @@
       name: middleware_monitoring_config
       tasks_from: upgrade_alert_manager_email.yml
 
-#Update product version (should always be last)
+  #Update product version (should always be last)
+  - name: Update webapp version info
+    include_role:
+      name: webapp
+      tasks_from: update_version_info.yml
+
```
Comment (mikenairn) [inline scripts/upgrade.template.yml:71]: Will prevent the webapp version issue happening again the next time we create a release https://issues.redhat.com/browse/INTLY-10083

**Comment by R-Lawton** [comment]
:eyes:

**Comment by mikenairn** [comment]
Added 20aa87feb92f43f4412d7be60724594087d8104d, although it's not required on master, it might be useful if we ever need to update the probes again.

**Pull request merged**